flow/project.py

```diff
@@ -800,13 +843,13 @@ def __repr__(self):
                    directives=self.operation_directives,
                    options=self.options)
 
-    def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
-        """Eligible, when at least one BaseFlowOperation is eligible.
+    def eligible(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
+        r"""Eligible, when at least one BaseFlowOperation is eligible.
 
-        :param job:
-            A :class:`signac.Job` from the signac workspace.
-        :type job:
-            :class:`signac.Job`
+        :param \*jobs:
+            The list of jobs that the :class:`~.JobOperation` is based on.
+        :type \*jobs:
+            list of :class:`signac.Job`
```

Comment:
Is user facing, and the API changes. Once again, this may be unavoidable but noting it.